samples: peripherals: add PPI sample (#645)
nrfconnect/sdk-nrf-bm: samples/peripherals/ppi/README.rst

.. _bm_ppi_sample:

> PPI - Programmable Peripheral Interconnect

Reviewer: ```suggestion
Programmable Peripheral Interconnect - PPI
```

           - S115
           - bm_nrf54l15dk/nrf54l10/cpuapp/s115_softdevice/mcuboot
         * - `nRF54L15 DK`_ (emulating nRF54L05)
           - PCA10156
           - S115
           - bm_nrf54l15dk/nrf54l05/cpuapp/s115_softdevice/mcuboot
         * - `nRF54L15 DK`_
           - PCA10156
           - S145
           - bm_nrf54l15dk/nrf54l15/cpuapp/s145_softdevice/mcuboot
         * - `nRF54L15 DK`_ (emulating nRF54L10)
           - PCA10156
           - S145
           - bm_nrf54l15dk/nrf54l10/cpuapp/s145_softdevice/mcuboot
         * - `nRF54L15 DK`_ (emulating nRF54L05)
           - PCA10156
           - S145
           - bm_nrf54l15dk/nrf54l05/cpuapp/s145_softdevice/mcuboot

Overview
********

The sample sets up an LED to be toggled on the trigger of a timer compare event.

User interface
**************

LED 0:
   Lit when the device is initialized.

LED 1:
   Blinking when the timer is running.

Building and running
********************

This sample can be found under :file:`samples/peripherals/ppi/` in the |BMshort| folder structure.

For details on how to create, configure, and program a sample, see :ref:`getting_started_with_the_samples`.

Testing
=======

> You can test this sample by performing the following steps:

Reviewer: ```suggestion
To test this sample, complete the following steps:
```

Author: This is the same text as in other samples. If this needs a change, it should be taken for all samples in a separate PR. Leaving as is for now.

.. _bm_ppi_sample:

Programmable Peripheral Interconnect (PPI)
##########################################

.. contents::
   :local:
   :depth: 2

The Programmable Peripheral Interconnect (PPI) sample demonstrates how to configure and use PPI with |BMlong|.

Requirements
************

The sample supports the following development kits:

> 

Author: We don't use that in BM. We use the tabs and tables below instead.

.. tabs::

   .. group-tab:: Simple board variants

      The following board variants do **not** have DFU capabilities.

      .. list-table::
         :header-rows: 1

         * - Hardware platform
           - PCA
           - SoftDevice
           - Board target
         * - `nRF54L15 DK`_
           - PCA10156
           - S115
           - bm_nrf54l15dk/nrf54l15/cpuapp/s115_softdevice
         * - `nRF54L15 DK`_ (emulating nRF54L10)
           - PCA10156
           - S115
           - bm_nrf54l15dk/nrf54l10/cpuapp/s115_softdevice
         * - `nRF54L15 DK`_ (emulating nRF54L05)
           - PCA10156
           - S115
           - bm_nrf54l15dk/nrf54l05/cpuapp/s115_softdevice

   .. group-tab:: MCUboot board variants

>       The following board variants have DFU capabilities.

Author: Fixed, will open separate PR to align other samples.
